expando/node_container.c

```diff
@@ -81,7 +81,13 @@ int node_container_render(const struct ExpandoNode *node,
   }
   else
   {
-    buf_addstr(buf, buf_string(tmp));
+    struct Buffer *tmp2 = buf_pool_get();
+
+    total_cols = format_string(tmp2, 0, max_cols, JUSTIFY_LEFT, ' ',
+                               buf_string(tmp), buf_len(tmp), true);
+
+    buf_addstr(buf, buf_string(tmp2));
+    buf_pool_release(&tmp2);
   }
```

Comment:
Would it make sense to move the `buf_pool_get`, `buf_addstr`, and `buf_pool_release` out of the if/else block?